_data/rules.yml

```diff
@@ -17,6 +17,9 @@ required:
 - description: You must change the name of the software if you modify it.
   label: Rename
   tag: rename
+- description: You must change the name of the software if you modify it and any Reserved Font Names are declared.
```

Comment:
Is it possible to ELI5 what a "Reserved Font Name" is within the description? Assume the reader has never heard of OFL or Font Licensing before. What does this mean?